public/examples.html

```diff
@@ -0,0 +1,432 @@
+<!DOCTYPE html>

```

Comment:
We might place this under kontent-ai.github.io or wee need to add the build to GitHub pages, so that we have a link to publish version of the examples.